src/algo/thermal-loop/l500-thermal-loop.h

```diff
@@ -0,0 +1,81 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2020 Intel Corporation. All Rights Reserved.
+
+#pragma once
+
+#include <vector>
+#include "../../types.h"
+
+namespace librealsense {
+namespace algo {
+namespace thermal_loop {
+namespace l500 {
+
+#pragma pack( push, 1 )
+
+// This struct based on RGB_Thermal_Info_CalibInfo table.
+// The table contains 29 equally spaced bins between min & max temperature.
+// Center of each bin has a set of 4 transformation parameters.
+// The transformation maps a point in the RGB image in a given temperature to its expected
+// Location in the temperature in which the RGB module was calibrated.
+// Reference on:
+// https://rsconf.intel.com/display/L500/0x317+RGB+Thermal+Table
+
+
+class thermal_calibration_table
+{
+public:
+    static const int id;
+    
+    struct thermal_table_header
+    {
+        float min_temp;
+        float max_temp;
+        float reference_temp;  // not used
+        float valid;           // not used
+    };
+
+    struct temp_data
+    {
+        float scale;
+        float sheer; // parameters which affect offset that are not in use
+        float tx;
+        float ty;
+    };
+
+    int _resolution;
```

Comment:
`// number of bins *between* min and max, i.e. bin-size=(max-min)/(resolution+1)`

Reply:
done